src/modules/auth/index.js

```diff
@@ -170,6 +173,27 @@ export default class AuthModule extends Module {
 		});
 	}
 
+	logoutOnExpiredSession (that, al=false) {
+		const difference = moment(that.UserInfo.exp).valueOf() - moment.now();
+		let alert = al;
+
+		if (!that.UserInfo?.exp) {
+			setTimeout(that.logoutOnExpiredSession, 1000, that);
+		}
+
+		if (difference <= 0) {
+			that.logout();
```

Comment:
Nope, don't do `logout`, it will be problematic for users.

Reply:
Okay, then I should change this to second notification that session has expired?

Reply:
yes